twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/ui/common/adapters/TweetsAdapter.kt

```diff
@@ -190,4 +187,14 @@ class TweetsAdapter(
             }
         }
     }
+
+    companion object {
+        val TweetsDiffCallback = object : DiffUtil.ItemCallback<Status>() {
+            override fun areItemsTheSame(oldItem: Status, newItem: Status): Boolean =
+                oldItem.id == newItem.id
+
+            override fun areContentsTheSame(oldItem: Status, newItem: Status): Boolean =
+                oldItem.id == newItem.id
```

Comment:
When you override this method you need to compare the items content. You can just do `oldItem == newItem` or `oldItem.equals(newItem)`.